backends/imgui_impl_metal.mm

```diff
@@ -193,7 +361,7 @@ - (instancetype)initWithBuffer:(id<MTLBuffer>)buffer
     if ((self = [super init]))
     {
         _buffer = buffer;
-        _lastReuseTime = [NSDate date].timeIntervalSince1970;
+        _lastReuseTime = GetMachAbsoluteTimeInSeconds();
```

Comment:
Does change to timing function fix anything, or is it just a "good practice" change?

Reply:
Was just good practice, as `[NSDate data].timeIntervalSince1970` is a bit slower (showed up in profiling).